command/tools.go

```diff
@@ -1,52 +1,80 @@
 package command
 
 import (
+	"change-status-go/sentence"
 	"fmt"
 	"github.com/bwmarrin/discordgo"
 	"log"
 	"strings"
 )
 
+const (
+	errorBadRequest = `HTTP 400 Bad Request, {"nick": ["Must be 32 or fewer in length."]}`
+	errorForbidden = `HTTP 403 Forbidden, {"message": "Missing Permissions", "code": 50013}`
+	prefix = "%"
+)
+
 func OnMessageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
-	c, err := s.State.Channel(m.ChannelID)
-	if err != nil {
-		log.Println("Error getting channel: ", err)
+	c, Err := s.State.Channel(m.ChannelID)
+	if Err != nil {
+		log.Println("Error getting channel: ", Err)
 		return
 	}
-
-	switch {
-		case strings.HasPrefix(m.Content, fmt.Sprintf("%s", hello)):
-			sendMessage(s, c, helloWorld)
-
-		case strings.HasPrefix(m.Content, fmt.Sprintf("%s", usage)):
-			sendMessage(s, c, usageSentence)
-
-		case strings.HasPrefix(m.Content, fmt.Sprintf("%s", callName)):
-			sendMessage(s, c, m.Member.Nick)
-
-		case strings.HasPrefix(m.Content, fmt.Sprintf(uid)):
+	if !strings.HasPrefix(m.Content, prefix) {
+		return
+	}
+	commandName := strings.Split(m.Content, " ")[0][1:]
```

Comment:
Prefixの変更に備えて、こうしたほうがいいかもしれない？

```suggestion
	commandName := strings.Split(m.Content, " ")[0][len(prefix):]
```